### PR TITLE
dev-python/weasyprint: version bump and cleanup

### PR DESCRIPTION
**Pull request by epsilon-0**

### CODE REVIEW

File: dev-python/weasyprint/weasyprint-52.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+HOMEPAGE="https://weasyprint.org https://github.com/Kozea/WeasyPrint"
+MY_PN="WeasyPrint"
+MY_P="${MY_PN}-${PV}"
+SRC_URI="mirror://pypi/${MY_P:0:1}/${MY_PN}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="jpeg tiff"
```
Comment (juippis) [inline dev-python/weasyprint/weasyprint-52.2.ebuild:21]: I know this wasn't added by you, but these USE flags make no difference in this ebuild. Just force them on or off with gdk-pixbuf below, whatever is required by upstream.

File: dev-python/weasyprint/weasyprint-52.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	>=dev-python/cairocffi-0.9[${PYTHON_USEDEP}]
+	>=dev-python/cffi-0.6:=[${PYTHON_USEDEP}]
+	>=dev-python/cssselect2-0.1[${PYTHON_USEDEP}]
+	>=dev-python/html5lib-0.999999999[${PYTHON_USEDEP}]
+	>=dev-python/pyphen-0.9.1[${PYTHON_USEDEP}]
+	>=dev-python/tinycss2-1.0.0[${PYTHON_USEDEP}]
+	>=media-gfx/cairosvg-2.4.0[${PYTHON_USEDEP}]
+	>=x11-libs/cairo-1.15.4
+	x11-libs/gdk-pixbuf[jpeg?,tiff?]
+	x11-libs/pango:0/0
```
Comment (juippis) [inline dev-python/weasyprint/weasyprint-52.2.ebuild:35]: I know this wasn't added by you either, but it seems suspicious. I doubt it's needed.

**Review by juippis — COMMENTED** [review]
```
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/dev-python/weasyprint-52.2/work/WeasyPrint-52.2 ...
>>> Source configured.
>>> Compiling source in /var/tmp/portage/dev-python/weasyprint-52.2/work/WeasyPrint-52.2 ...
 * python3_7: running distutils-r1_run_phase distutils-r1_python_compile
python3.7 setup.py build -j 16
WARNING: The wheel package is not available.
/usr/bin/python3.7: No module named pip
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/setuptools/installer.py", line 75, in fetch_build_egg
    subprocess.check_call(cmd)
  File "/usr/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/usr/bin/python3.7', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/tmp/portage/dev-python/weasyprint-52.2/temp/tmp19mdl3s1', '--quiet', 'pytest-runner']' returned non-zero exit status 1.
```

After installing dev-python/wheel, it turned into:

```
>>> Compiling source in /var/tmp/portage/dev-python/weasyprint-52.2/work/WeasyPrint-52.2 ...
 * python3_7: running distutils-r1_run_phase distutils-r1_python_compile
python3.7 setup.py build -j 16
/usr/bin/python3.7: No module named pip
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/setuptools/installer.py", line 75, in fetch_build_egg
    subprocess.check_call(cmd)
  File "/usr/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/usr/bin/python3.7', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/tmp/portage/dev-python/weasyprint-52.2/temp/tmp45bm0dp2', '--quiet', 'pytest-runner']' returned non-zero exit status 1.
```

You can clearly see the call for pip here:
`distutils.errors.DistutilsError: Command '['/usr/bin/python3.8', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/tmp/portage/dev-python/weasyprint-52.2/temp/tmps6_szi80'`

After installing pip, 
```
>>> Compiling source in /var/tmp/portage/dev-python/weasyprint-52.2/work/WeasyPrint-52.2 ...
 * python3_7: running distutils-r1_run_phase distutils-r1_python_compile
python3.7 setup.py build -j 16
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fe2983dbe50>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/pytest-runner/
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fe2983f9210>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/pytest-runner/
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fe2983f9550>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/pytest-runner/
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fe2983f9890>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/pytest-runner/
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0x7fe2983f9bd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/pytest-runner/
ERROR: Could not find a version that satisfies the requirement pytest-runner
ERROR: No matching distribution found for pytest-runner
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/setuptools/installer.py", line 75, in fetch_build_egg
    subprocess.check_call(cmd)
  File "/usr/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/usr/bin/python3.7', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/tmp/portage/dev-python/weasyprint-52.2/temp/tmpebz4y6_n', '--quiet', 'pytest-runner']' returned non-zero exit status 1.
```
So the tests don't work for me.

**Comment by ConiKost** [comment]
Seems, someone else was faster, as we have newer versions in tree, so closing here.

**Pull request closed without merging**